doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -122,6 +122,14 @@ given to L<OSSL_STORE_open_file(3)>, the returned name will be the path of
 each object, so if C</foo/bar> was given and that path has the file
 C<cookie.pem>, the name C</foo/bar/cookie.pem> will be returned.
 
+The returned URI is considered canonical and must be unique and permanent
+for the storage where the object (or collection of objects if that's what
```

Comment:
Why do we even need anything more in the parenthetical than "(or collection of objects)"?